src/screens/Notifications/ConfirmEmailScreen.tsx

```diff
@@ -192,26 +173,27 @@ class ConfirmEmailScreen extends Component<Props, State> {
           </Text>
         </View>
         <View style={styles.inputItemContainer}>
-          <CodeInput value={this.state.code} onTextChange={this.onChange} isError={!!this.state.error} />
-          <Text style={styles.error}>{this.state.error}</Text>
+          <CodeInput value={this.state.code} onTextChange={this.onChange} isError={!!notificationError} />
+          <Text style={styles.error}>{notificationError}</Text>
```

Comment:
`{notificationError && <Text style={styles.error}>{notificationError}</Text>}` maybe? So we ca avoid rendering it at all? Not sure about styles, but if somehow it gets height in some future implementation changes, it may render empty and mess the view

Reply:
same with this.state.error

Reply:
ok will add, there is no state error

Reply:
true, it s been removed 👀

Reply:
done